crates/cargo-platform/src/lib.rs

```diff
@@ -115,30 +116,18 @@ impl Platform {
                         check_cfg_expr(e, warnings, path);
                     }
                 }
+                CfgExpr::True | CfgExpr::False => {}
                 CfgExpr::Value(ref e) => match e {
                     Cfg::Name(name) | Cfg::KeyPair(name, _) => {
                         if !name.raw && KEYWORDS.contains(&name.as_str()) {
-                            if name.as_str() == "true" || name.as_str() == "false" {
-                                warnings.push(format!(
-                                    "[{}] future-incompatibility: the meaning of `cfg({e})` will change in the future\n \
-                                     | Cargo is erroneously allowing `cfg(true)` and `cfg(false)`, but both forms are interpreted as false unless manually overridden with `--cfg`.\n \
-                                     | In the future these will be built-in defines that will have the corresponding true/false value.\n \
-                                     | It is recommended to avoid using these configs until they are properly supported.\n \
-                                     | See <https://github.com/rust-lang/rust/issues/131204> for more information.\n \
-                                     |\n \
-                                     | help: use raw-idents instead: `cfg(r#{name})`",
-                                    path.display()
-                                ));
-                            } else {
-                                warnings.push(format!(
-                                    "[{}] future-incompatibility: `cfg({e})` is deprecated as `{name}` is a keyword \
-                                     and not an identifier and should not have have been accepted in this position.\n \
-                                     | this was previously accepted by Cargo but is being phased out; it will become a hard error in a future release!\n \
-                                     |\n \
-                                     | help: use raw-idents instead: `cfg(r#{name})`",
-                                     path.display()
-                                ));
-                            }
+                            warnings.push(format!(
```

Comment:
Do we not have a test for this remaining future-incompat warning?

Reply:
I don't think so, the keywords list currently only contains `true` and `false`.

https://github.com/rust-lang/cargo/blob/a6c604d1b8a2f2a8ff1f3ba6092f9fda42f4b7e9/crates/cargo-platform/src/cfg.rs#L47-L53